scripts/zones/Sealions_Den/helpers/One_to_be_Feared.lua

```diff
@@ -0,0 +1,141 @@
+-----------------------------------
+-- Global file for "One to be Feared" Battlefield
+-----------------------------------
+local ID = require("scripts/zones/Sealions_Den/IDs")
+require("scripts/globals/titles")
+-----------------------------------
+xi = xi or {}
```

Comment:
This file should probably return an object and not be attached to the xi global, for as specific as it is